graphql-spring-boot-autoconfigure/src/main/java/graphql/kickstart/autoconfigure/web/servlet/AsyncServletProperties.java

```diff
@@ -1,14 +1,25 @@
 package graphql.kickstart.autoconfigure.web.servlet;
 
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
 import lombok.Data;
+import lombok.NonNull;
 import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.convert.DurationUnit;
 
 @Data
 @ConfigurationProperties(prefix = "graphql.servlet.async")
 public class AsyncServletProperties {
 
+  public static final Duration DEFAULT_TIMEOUT = Duration.ofSeconds(30);
+
   private boolean enabled = true;
-  private long timeout = 30000;
+  /**
+   * Asynchronous execution timeout. If a duration suffix is not specified, millisecond will be used.
+   */
+  @NonNull
```

Comment:
I would drop `@NonNull` for now, and return to the validation part in a separate PR.

Reply:
I've dealt with it.